services/wallet/cryptocompare.go

```diff
@@ -0,0 +1,128 @@
+package wallet
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type Coin struct {
+        Id                      string                 `json:"Id"`
+        Name                    string                 `json:"Name"`
+        Symbol                  string                 `json:"Symbol"`
+        Description             string                 `json:"Description"`
+        TotalCoinsMined         float64                `json:"TotalCoinsMined"`
+        AssetLaunchDate         string                 `json:"AssetLaunchDate"`
+        AssetWhitepaperUrl      string                 `json:"AssetWhitepaperUrl"`
+        AssetWebsiteUrl         string                 `json:"AssetWebsiteUrl"`
```

Comment:
lint is failing because of `AssetWebsiteUrl` expecting `AssetWebsiteURL`
same for `AssetWhitepaperUrl`

Reply:
done!